docs/pages/User/Installation/aks.md

```diff
@@ -0,0 +1,161 @@
+  the service will be exposed with a [NodePort Service](https://kubernetes.io/docs/concepts/services-networking/service/#nodeport)
+  instead of an [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/)
+* `LIQO_AUTHSERVER_PORT` -> the port where to access the Auth Service (defaults to `443`)
+
+#### How can I know those variable values in AKS?
+
+Some variables are the same for each AKS cluster.
+
+| Variable               | Value                          | Notes                                  |
+| ---------------------- | ------------------------------ | -------------------------------------- |
+| `POD_CIDR`             | 10.244.0.0/16                  |                                        |
+| `SERVICE_CIDR`         | 10.0.0.0/16                    |                                        |
+| `LIQO_INGRESS_CLASS`   | addon-http-application-routing | If you enabled the built-in AKS plugin |
+| `LIQO_APISERVER_PORT`  | 443                            |                                        |
+| `LIQO_AUTHSERVER_PORT` | 6443                           |                                        |
+
+The other two values can be found in the Azure Portal.
+
+The `LIQO_APISERVER_ADDR` con be found in our cluster overview as __API server address__
+
+![](/images/install/aks/06.png)
+
+The `LIQO_AUTHSERVER_ADDR` is where the Liqo Auth Service will be reachable, so we have to export some hostname that we
+can manage. If you are using the AKS HTTP Application Routing a DNS zone should be available, so you can create a subdomain
+on it.
+
+![](/images/install/aks/07.png)
+
+In this case a viable hostname would be `auth.f83c28d9ce1449b2bb45.westeurope.aksapp.io`
+
+#### Install Liqo
+
+After this configuration step we can [run the Liqo one line installer](/user/gettingstarted/install/#default-install)
+as usual.
+
+#### Example
+
+```bash
+export POD_CIDR=10.244.0.0/16
+export SERVICE_CIDR=10.0.0.0/16
+export LIQO_INGRESS_CLASS=addon-http-application-routing
+export LIQO_APISERVER_PORT=443
+export LIQO_APISERVER_ADDR=liqo-dns-6f1bc41d.hcp.westeurope.azmk8s.io
+export LIQO_AUTHSERVER_ADDR=auth.f83c28d9ce1449b2bb45.westeurope.aksapp.io
+```
+
+```bash
+curl -sL https://get.liqo.io | bash
+```
+
+## Check that Liqo is Running
+
+Wait that all Liqo pods are up and running
+
+```bash
+kubectl get pods -n liqo
+```
+
+![](/images/install/aks/08.png)
+
+### Access the cluster configurations
+
+You can get the cluster configurations from the Auth Service endpoint to check that this service has been correctly deployed
+
+```bash
+curl --insecure https://auth.f83c28d9ce1449b2bb45.westeurope.aksapp.io/ids
+```
+
+```json
+{"clusterId":"4e33e2dc-b2e3-4052-9a74-d18f6d8cdab2","clusterName":"LiqoCluster5852","guestNamespace":"liqo"}
+```
+
+Congratulations! Liqo is now up and running on your AKS cluster, you can now peer with other Liqo instances!
```

Comment:
I would add a few commands explaining how we can use the aforementioned data to enable a peering on the other cluster.